wallet/src/main/scala/org/bitcoins/wallet/models/AddressTagDAO.scala

```diff
@@ -106,7 +132,8 @@ case class AddressTagDAO()(implicit
   class AddressTagTable(t: Tag)
       extends Table[AddressTagDb](t, "wallet_address_tags") {
 
-    def address: Rep[BitcoinAddress] = column[BitcoinAddress]("address")
+    def address: Rep[BitcoinAddress] =
+      column[BitcoinAddress]("address", O.PrimaryKey)
```

Comment:
Does the addition of this index require a migration?

Reply:
No, was already in the migrations, the slick just did not match the db